### PR TITLE
Require minimum return data from EVMScript executors to prevent selfdestructs

### LINKED ISSUE #250: Prevent EVMScript runners from self destructing apps

### PR DESCRIPTION
**Pull request by izqui**
Close #250

### CODE REVIEW

**Comment by izqui** [comment]
CI doing stupid stuff. Tests are passing locally

**Comment by bingen** [comment]
Yes, I had the exact same problem a while ago with one of my PRs. Following your advice and just restarting it did the trick ;-)

File: test/evm_script.js [hunk 1]
```diff
@@ -279,5 +290,29 @@ contract('EVM Script', accounts => {
...
+                dyingExecutor = await MockDyingScriptExecutor.new()
+                await reg.addScriptExecutor(dyingExecutor.address, { from: boss })
+            })
+
+            it('registers new executor', async () => {
+                assert.equal(await reg.getScriptExecutor('0x00000004'), dyingExecutor.address)
+            })
+
+            it('executes mock executor', async () => {
+                await executor.execute('0x00000004')
```
Comment (sohkai) [inline test/evm_script.js]: Maybe this could check that the result is actually `bytes(1)`?

Comment (izqui) [reply test/evm_script.js]: Executor.execute doesn't return anything, it would require some changes

Comment (izqui) [reply test/evm_script.js]: Executor.execute doesn't return anything, it would require some changes

Comment (sohkai) [reply test/evm_script.js]: Ah damn, ok. Let's not worry about it then.

**Pull request merged**